runtime/runtime-params-estimator/test-contract/res/near_evm.wasm

```diff
@@ -0,0 +1 @@
+../../../../../near-evm/res/near_evm.wasm
```

Comment:
That's need to be included into the nearcore

Reply:
Yeah I planned on writing a script which would pull it at the same level as nearcore's parent directory.